src/components/WaitingRoom/index.js

```diff
@@ -134,28 +147,61 @@ export default function WaitingRoom({ location }) {
     setLocalVideo(e.target.checked);
   }, []);
 
+  const handleChangeBackgroundBlur = React.useCallback(async () => {
```

Comment:
@javiermolsanz looks good. Can you only add a try-catch block?

Reply:
sure thing